pmd/.htaccess

```diff
@@ -7,29 +7,34 @@ RewriteCond %{HTTP_ACCEPT} text/html [OR]
 RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml 
 RewriteRule ^co(.*)$ https://materialdigital.github.io/core-ontology/index-en.html [R=303,L]
 RewriteRule ^glass-ontology(.*)$ https://materialdigital.github.io/glasdigital-ontology/index-en.html [R=303,L]
+RewriteRule ^tto(.*)$ https://materialdigital.github.io/application-ontologies/tto/index-en.html [R=303,L]
```

Comment:
I don't think this file is doing what you want, including these added rules.  The `RewriteCond` lines only effect the *next* `RewriteRule`.  They are not stateful for all following rules.  Please test and confirm.

For instance, requesting `application/ld+json` for `glass-ontology` will return the first matching rule, which is the HTML one above.
```
$ curl -sI -H "Accept: application/ld+json" https://w3id.org/pmd/glass-ontology | grep Location
Location: https://materialdigital.github.io/glasdigital-ontology/index-en.html
```

You may want to use the skip flag `S=N` "goto" trick.  There are some other hacks people have used too if you search around for multiple `RewriteRules` for one `RewriteCond`.
https://httpd.apache.org/docs/2.4/rewrite/flags.html#flag_s